firebase-firestore/src/main/java/com/google/firebase/firestore/remote/IncomingStreamObserver.java

```diff
@@ -25,7 +25,10 @@
   /** A message was received on the stream. */
   void onNext(RespT response);
 
-  /** The stream is "ready" (What the hell does that mean?!). */
+  /** The stream is open and able to accept messages. */
+  void onOpen();
+
+  /** The stream is "ready". */
```

Comment:
You seem to have forgotten part of the comment. 😛 

Perhaps add a link to https://grpc.github.io/grpc-java/javadoc/io/grpc/ClientCall.Listener.html#onReady-- and/or say "(additional messages can be sent to the stream without incurring excessive buffering)"

We could also just remove it since we don't use it for anything...  though I guess the log message might be meaningful.

Reply:
I may have deliberately removed it since I found it kind of quirky :/ I just went one step further and removed the callback. If you think we need this logging (I am thinking we don't), we could also log in GrpcCallProvider.